rust/Cargo.lock

```diff
@@ -4708,16 +4708,17 @@ dependencies = [
 
 [[package]]
 name = "quinn"
```

Comment:
Supposedly, this one is being pulled in by `reqwest`. But we already disable all its features and it shouldn't so I don't quite understand what is happening here.